ibis/omniscidb/tests/test_client.py

```diff
@@ -154,6 +155,57 @@ def test_explain(con, alltypes):
     con.explain(alltypes)
 
 
+@pytest.mark.parametrize(
+    'filename',
+    [
+        "/omnisci/test_read_csv.csv",
+        pathlib.Path("/omnisci/test_read_csv.csv"),
+    ],
+)
+def test_read_csv(con, filename):
+    t_name = "test_read_csv"
+
+    con.drop_table(t_name, force=True)
+    schema = ibis.schema(
+        [
+            ('index', 'int64'),
+            ('Unnamed__0', 'int64'),
+            ('id', 'int32'),
+            ('bool_col', 'bool'),
+            ('tinyint_col', 'int16'),
+            ('smallint_col', 'int16'),
+            ('int_col', 'int32'),
+            ('bigint_col', 'int64'),
+            ('float_col', 'float32'),
+            ('double_col', 'double'),
+            ('date_string_col', 'string'),
+            ('string_col', 'string'),
+            ('timestamp_col', 'timestamp'),
+            ('year_', 'int32'),
+            ('month_', 'int32'),
+        ]
+    )
+    con.create_table(t_name, schema=schema)
```

Comment:
use a fixture to handle resource creation & destruction rather than using a try/finally (you can do that in the fixture itself)